ethereum/api/src/main/resources/schema.graphqls

```diff
@@ -114,12 +114,12 @@ input BlockFilterCriteria {
   # contained topics.
   #
   # Examples:
-  #  - [] or nil          matches any topic list
+  #  - [] or null         matches any topic list
   #  - [[A]]              matches topic A in first position
-  #  - [[], [B]]          matches any topic in first position, B in second position
+  #  - [[null], [B]]      matches any topic in first position, B in second position
   #  - [[A], [B]]         matches topic A in first position, B in second position
   #  - [[A, B]], [C, D]]  matches topic (A OR B) in first position, (C OR D) in second position
-    topics: [[Bytes32!]!]
+    topics: [[Bytes32]!]
```

Comment:
The ! means not-null
So this change means the inner list/s can now contain null values. But the list still can't _be_ null
https://hasura.io/blog/graphql-nulls-cheatsheet/

Reply:
thanks @shemnon - I'll take another look

Reply:
IMO using an empty array [] is much safer than using a one-null-element array [null] due to the "billion dolar mistake". The natural evolution of programming languages is toward getting rid of nulls since old languajes/libraries/frameworks make it very easy to generate false ones. Some moderns languages interacting with Besu do not even include the possibility to deal with nulls (vlang, Haskell, OCaml, Rust, *ML, Swift, ...).

The null "inside" [null] can be the desired one, or it can be the result of a parsing-bug, a non initialized javascript variable or another non-desired error. Detecting a null must raise an error to "fail fast". Code can not get responsible for guessing whether an external (non-controlled) input null was the desired one.

REFs:
* https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare/
* https://en.wikipedia.org/wiki/Fail-fast

Reply:
thanks for the feedback folks, have updated the PR